packages/oauth/src/install-provider.ts

```diff
@@ -27,7 +27,7 @@ import defaultRenderHtmlForInstallPath from './default-render-html-for-install-p
 import { InstallPathOptions } from './install-path-options';
 
 /**
- * InstallProvider Class. Refer to InsallProviderOptions interface for the details of constructor arguments.
+ * InstallProvider Class. Refer to InstallProviderOptions interface for the details of constructor arguments.
```

Comment:
Thanks for detecting errors in the comments!

Reply:
No worries, my IDE does most of the work :)